### DISCUSSION TITLE
Motion Zones

### DISCUSSION THREAD

**Discussion by segfaultex** [briis/unifiprotect#206] [Q&A]
Is there any way to read what zone motion happened in?
I've got an automation that notifies of motion but I would like to filter out vehicles detected from one of the smart detection zones.

**Comment by briis** [comment]
No, unfortunately that information is not send with the event: Below is and example of what we get, and the motion zone being triggered is not part of that. But you never know, it might be there in the future, so I'll keep an eye out for it.

```
    Smart Detect Event Add:
    {'action': 'add', 'newUpdateId': '032615bb-910d-41bf-8710-b04959f24455', 'modelKey': 'event', 'id': '5fb0c89003085203870013d0'}
    {'type': 'smartDetectZone', 'start': 1605421197481, 'score': 98, 'smartDetectTypes': ['person'], 'smartDetectEvents': [], 'camera': '5f9f43f102f7d90387004da5', 'partition': None, 'id': '5fb0c89003085203870013d0', 'modelKey': 'event'}

    Camera Motion Start (event):
    {'action': 'add', 'newUpdateId': '25b1142a-2d0d-4b85-b97e-401b03dd1f0b', 'modelKey': 'event', 'id': '5fb0c90603455203870013d7'}
    {'type': 'motion', 'start': 1605421315759, 'score': 0, 'smartDetectTypes': [], 'smartDetectEvents': [], 'camera': '5e539ed503617003870003ed', 'partition': None, 'id': '5fb0c90603455203870013d7', 'modelKey': 'event'}
```

**Reply by DuelCoreF18** [reply]
I think the other 2 post and myself are all after smart detection over regular motion detection. Looks like it comes through with the smartDetectTypes field. I think I follow where the OP is going with this as motion and smart detection are set up as separate zones.

**Reply by briis** [reply]
See my reply in #171
